primer/brand
# Generate fullPage screenshots

## Summary

Changes the generated screenshots to capture the full page. This means that the virtual screens the screenshots are taken on have infinite height to capture the entire page.

## List of notable changes:



## What should reviewers focus on?

check the first commit that I made the correct change to capture the screenshots and didn't miss any thing.

## Steps to test:

You can open storybook for this change and see all the changes. They are especially notable in the "Recipes" section

## Supporting resources (related issues, external links, etc):

- https://playwright.dev/docs/screenshots#full-page-screenshots
-

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">



 </td>
<td valign="top">
![image](https://github.com/primer/brand/assets/12260694/414ee592-478a-4948-af7b-431012b85bde)


![image](https://github.com/primer/brand/assets/12260694/8e6b303a-bf28-45bc-a6a3-163c00c41477)

</td>
</tr>
</table>
